inst/rmarkdown/templates/tufte_book/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,264 @@
+---
+title: "Tuftish Book"
+author: "John Smith"
+date: "August 13th, 2014"
+bibliography: "knitcitations.bib"
+output: 
+  tufte_book:
+    latex_engine: lualatex
+    toc: true
+    toc_depth: 3
+---
+
+```{r setup,echo=FALSE,message=FALSE,cache=FALSE,include=TRUE}
+# Its a good idea to start any rmarkdown document with a block like this one to set knit defaults, setup the bibliography, and so forth.
+library(knitcitations)
+library(knitr)
+library(xtable)
+library(ggplot2)
+library(papeR)
+library(pander)
+cite_options(citation_format = "pandoc",
+  					 cite.style= "authortitle",
+						 style="citation",
+						 hyperlink="to.doc",
+						 check.entries=FALSE)
+
+opts_chunk$set(   echo = FALSE,
```

Comment:
I think it's fine to illustrate the setup chunk, but I don't think we should reset all of this default behavior (e.g. adding caching). Perhaps just a one-liner with a couple options or even a commented out version. If we really think these options are important then we should actually define them as part of the format.